src/System.Private.ServiceModel/tests/Scenarios/Http/TransportChannel/BasicBindings/BasicHttpBindingTests.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.ServiceModel;
+using System.ServiceModel.Channels;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using TestTypes;
+using Xunit;
+
+public static class Http_TransportChannel_BasicBindings_BasicHttpBindingTests
+{
+
+    [Fact]
+    [OuterLoop]
+    public static void DefaultSettings_EchoString_RoundTrips()
```

Comment:
I think the intent was to say "it round trips successfully" so perhaps we need a different naming pattern. RoundTrip_String_Using_Defaults?